snippets/language-html.cson

```diff
@@ -1,4 +1,4 @@
-'.text.html, .text.html .meta.scope.between-tag-pair, .text.html .punctuation.tag.begin':
+'.text.html, .text.html .meta.scope.between-tag-pair, .text.html .punctuation.tag.begin, .source.gfm':
```

Comment:
Then how about adding snippets for elements in the whitelist that are not covered by the GFM syntax directly in language-gfm?